packages/page-pattern-modal/src/components/page-pattern-modal.tsx

```diff
@@ -131,7 +131,8 @@ class PagePatternModal extends Component< PagePatternModalProps, PagePatternModa
 	setPattern = ( name: string ) => {
 		// Track selection and mark post as using a pattern in its postmeta.
 		trackSelection( name );
-		this.props.savePatternChoice( name );
+		const { selectedCategory } = this.state;
```

Comment:
Might it be cleaner to have the category supplied directly via the `setPattern()` function? Looking it up via the state feels a bit roundabout. (Not a blocker.)

Reply:
Hum, we would need to add a new prop, the `PatternSelectorControl` component. And there, we would add the second argument. It seems too much back and forth, no? We don't have the category info in the PatternSelectorControl component.